NDB.Covid19/NDB.Covid19.Droid/Resources/layout/infection_status.xml

```diff
@@ -34,7 +34,7 @@
 	<ImageButton
 		android:layout_width="26dp"
 		android:layout_height="18dp"
-			android:layout_margin="16dp"
+		android:layout_margin="@dimen/defaultHorizontalMargin"
```

Comment:
This .xml file is changed significantly in my recent PR. So please await those changes and add yours afterwards

Reply:
conflicts resolved